inductiva/tasks/methods.py

```diff
@@ -89,14 +27,104 @@ def get_tasks_info(status: Optional[str] = None,
         }
 
         if status is not None:
-            query_params["status"] = TaskStatusCode(status)
+            query_params["status"] = models.TaskStatusCode(status)
 
         try:
             # Get User Tasks
-            api_response = api_instance.get_user_tasks(
-                query_params=query_params,)
+            resp = api_instance.get_user_tasks(
```

Comment:
With the description you provide below, why not `response_dict`?

Moreover, I don't really see a reason for this change, since it looked cleaner before and the output seems to be the same. Can you elaborate?

Reply:
not response_dict because resp is the urllib3 response object, not a dict. the reason for the change was to get rid of the annoying DynamicSchemas that the generated client uses when serializing. where not using the objects that the client generates so I'll rather just use the raw json data. at some point we'll probably serialize this to our own dataclass

Reply:
Alright, seems good! @david451lima look into this for your work, since that was also our concern at a certain point.